src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-duration;
 
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    transform: scale(1.2);
+    color: #34568b;
+  }
```

Comment:
3. [SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.

You should use hover inside . &__card. Look  like here:

 &:hover {
      transform: scale(1.2);
      color: #34568b;
    }